tests/test_main.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+import base64
+import os
+
+import aiohttp_jinja2
+import jinja2
+import pytest
+from aiohttp import web
+from aiohttp_session import setup as setup_session
+from aiohttp_session.cookie_storage import EncryptedCookieStorage
+from cryptography import fernet
+
+from blurb_it import __main__
+
+
+def create_app() -> web.Application:
+    fernet_key = fernet.Fernet.generate_key()
+    secret_key = base64.urlsafe_b64decode(fernet_key)
+
+    app = web.Application(
+        middlewares=[
+            __main__.middleware.error_middleware,
+            __main__.session_middleware(EncryptedCookieStorage(secret_key)),
+        ]
+    )
+
+    aiohttp_jinja2.setup(
+        app, loader=jinja2.FileSystemLoader(os.path.join(os.getcwd(), "templates"))
```

Comment:
```suggestion
        app, loader=jinja2.FileSystemLoader(pathlib.Path.cwd() / "templates")
```
or if jinja doesn't like path objects:
```suggestion
        app, loader=jinja2.FileSystemLoader(str(pathlib.Path.cwd() / "templates"))
```